examples/schneider_report.py

```diff
@@ -24,3 +24,8 @@
 outputText = template.render(group=group, project=project, build=build, testruns=testruns)
 with open('schneider_generated_report.html', 'w') as reportFile:
     reportFile.write(outputText)
+if os.getenv('TO_PDF'):
```

Comment:
This makes sense for the docker case, but for the pip package, it is not the case. Again, this will be revisited as optimisation progresses.

Reply:
you can remove this and add a shell script to the docker. This way you remove the depenency on wkhtml2pdf from pip package.

Reply:
@mwasilew there is no dependency on pip. It rather gets triggered upon docker run via -e to docker.

Reply:
It's a Dockerfile dependency